src/System Application/App/Guided Experience/src/Guided Experience/GuidedExperienceItemCleanup.Codeunit.al

```diff
@@ -0,0 +1,134 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.Environment.Configuration;
+
+using System.Apps;
+
+codeunit 1994 "Guided Experience Item Cleanup"
+{
+    InherentEntitlements = X;
+    InherentPermissions = X;
+    Permissions =
```

Comment:
internal

Reply:
Actually, just realized that I made it public on purpose... Note: all the procedures are either `local` or `internal`

Because we will need to access this codeunit to be able to schedule the Job Queue in BaseApp `Codeunit::"Guided Expereince Item Cleanup"`

Even thought Job Queue is planned to move to System App, we still need to do this for now